### ISSUE TITLE
Would someone like to take ownership of this project?

### ISSUE THREAD

**Issue by aisthesis** [GriffinAustin/pynance#33]
This is a very useful library that should be easy to maintain. Necessary maintenance would just be to make the necessary changes if providers of financial data evolve over time (possibly adding new ones, deprecating obsolete ones, changing code to correspond to new APIs, although `pandas` updates should be taking care of this. 

The library also offers some nice ML tools that I developed at first just for my own use.

Anyone actively using the library would be a good candidate to take it over. I haven't been using it for a couple of years, so don't even know firsthand any more if everything still works. It should, at least using the specified versions of everything. But I'm sure many of the dependencies already have new versions that should be supported.

I'll be happy to consult on any changes, but if anyone is still actively using the library, they would be well-served to take over the repo to keep it updated and perhaps even expand if you wish.

**Comment by prp1277** [comment]
@aisthesis I just stumbled across this after thinking "Pynance would be a great library name". 

I'm new to Python(and on mobile at the moment), but am very interested. I'll take a look when I get home tonight to see how I can help.

**Comment by prp1277** [comment]
I looked over the codebase and I think am way too new to python to take something like this on.

I have a few ideas as far as new endpoints to use, such as [iextrading](https://iexcloud.io/) & [alpaca](https://app.alpaca.markets/signup), but in terms of managing a library I would be in way over my head.

Learning `python` / `pip` / `<insert package manager here>` from a background in `npm` / `yarn` has been a real struggle (especially on Windows).

**Comment by GriffinAustin** [comment]
I've been using this library for quite some time now on my personal finance projects, and after taking a closer look at the codebase, I'd like to take this under my belt. I'm quite familiar with the libraries used as well as the data providers. I also have a few ideas and expansions for the ML tools.

I think @prp1277 will also bring some good modernization changes to the table. If you'd like to talk some more about your ideas, I think that'd be a good idea.

Let me know if this sounds like a good idea moving forward, @aisthesis, and if and/or how much you would like to continue to be a part of this awesome library!

**Comment by aisthesis** [comment]
I'd be delighted for you to take it under your belt. Let me know how I can help.  The first thing that comes to mind would be getting pip install to point to your fork rather than mine.

**Comment by GriffinAustin** [comment]
That sounds like a good plan. I think the best way to go about this would be to add me as an owner on PyPI; my username is [griffinaustin](https://pypi.org/user/griffinaustin/). Also, archiving this repository and linking to [my fork](https://github.com/GriffinAustin/pynance) in the description to show this one is deprecated would be a good thing to do.

As for moving forward with the project, I think we should start by updating the dependencies. This includes moving from matplotlib.finance, which is deprecated, to [mplfinance](https://github.com/matplotlib/mplfinance) as well as some others. I'll open an issue on my fork with more details so we can get started updating this project for 2020!

**Comment by aisthesis** [comment]
Hi Griffin, as you've no doubt observed, I haven't been active on here in a while. Would you still like me to add you as owner on PyPi?

> That sounds like a good plan. I think the best way to go about this would be to add me as an owner on PyPI; my username is [griffinaustin](https://pypi.org/user/griffinaustin/). Also, archiving this repository and linking to [my fork](https://github.com/GriffinAustin/pynance) in the description to show this one is deprecated would be a good thing to do.
> 
> As for moving forward with the project, I think we should start by updating the dependencies. This includes moving from matplotlib.finance, which is deprecated, to [mplfinance](https://github.com/matplotlib/mplfinance) as well as some others. I'll open an issue on my fork with more details so we can get started updating this project for 2020!

**Comment by aisthesis** [comment]
Feel free to PM me so I remember to do it this time.

**Comment by GriffinAustin** [comment]
> 
> 
> Feel free to PM me so I remember to do it this time.

Hello @aisthesis, I am still interested in taking up this project so that does include adding me as owner.

**Comment by aisthesis** [comment]
Attempting to transfer ownership, I got a message "GriffinAustin/pynance" already exists. So, it may not have been successful.

**Comment by GriffinAustin** [comment]
It appears successful. Just to be clear, the repo is staying as this one and not as my fork, correct? Also, how should we move forward with PyPI?

**Comment by GriffinAustin** [comment]
I deleted my fork if you want to give it another try. Moving forward with PyPI, do you want to add me as a maintainer? It seems that would be the easiest and best.

EDIT: It is showing me as collaborator so I assume we are good to go there.

**Comment by aisthesis** [comment]
It worked now adding you as actual owner. I'll presumably show up as collaborator.

Is your handle on PyPi also GriffinAustin? If so, you should have an invitation as collaborator.

**Comment by GriffinAustin** [comment]
Looks like we are good to go now! I look forward to helping revitalize this project.

**Comment by aisthesis** [comment]
Thank you, Griffin.

How about the documentation website? http://pynance.net/

**Comment by GriffinAustin** [comment]
I think it would be a good idea to migrate the documentation to readthedocs because we are currently using Sphinx.

**Comment by aisthesis** [comment]
Sounds good. The domain `pynance.net` is coming up for renewal. Is it ok for me to let it expire? Simply maintaining the domain doesn't cost much if there's a server you can point it to. The documentation currently appearing there was generated using Sphinx.

**Comment by GriffinAustin** [comment]
Yes, go ahead and let it expire and I will set up readthedocs with Sphinx

**Comment by aisthesis** [comment]
Just fyi: while the hosting of pynance.net will expire 3/2/2021, I have the domain until 3/2/22 in case you decide you'd like it in the meantime. Google Analytics has it currently with 135 users and around 25 hits daily.

**Comment by GriffinAustin** [comment]
Looks like I will also need to be added as a maintainer on PyPI test

**Comment by aisthesis** [comment]
I added you as maintainer on PyPI test.
